apps/fluent-tester/src/TestComponents/Chip/ChipTest.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+
+import { ChipDefault } from './ChipDefault';
+import { E2EChipTest } from './E2EChipTest';
+import { CHIP_TESTPAGE } from '../../../../E2E/src/Chip/consts';
+import type { TestSection, PlatformStatus } from '../Test';
+import { Test } from '../Test';
+
+const chipSections: TestSection[] = [
+  {
+    name: 'Default Chip',
+    testID: CHIP_TESTPAGE,
+    component: ChipDefault,
+  },
+];
+
+const e2eSections: TestSection[] = [
+  {
+    name: 'Chip E2E',
+    component: E2EChipTest,
+  },
+];
+
+export const ChipTest: React.FunctionComponent = () => {
+  const status: PlatformStatus = {
+    win32Status: 'Production',
```

Comment:
is chip already available for other platforms ? 

we should mark as backlog and only consider android if not.

Reply:
Updated